src/main/java/org/xstefank/verification/RepositoryFormatVerification.java

```diff
@@ -0,0 +1,17 @@
+package org.xstefank.verification;
+
+import org.xstefank.model.yaml.FormatConfig;
+
+public class RepositoryFormatVerification implements Verification {
+
+    private final String MESSAGE = "Bad repository format in configuration file";
+    private final String REGEX = "^[A-z,0-9]*./[A-z,0-9,-].*$";
```

Comment:
are you sure this regex is right? can you please include the link to the official rules for the repository naming conventions/rules?